scrapers/GotFilled.yml

```diff
@@ -64,10 +64,4 @@ xPathScrapers:
 
 driver:
   useCDP: true
-  cookies:
-    - Cookies:
-      - Name: gotfilled_warning_ok
-        Domain: gotfilled.com
-        Value: "1"
-        Path: /
```

Comment:
The "age > 18" prompt does not prevent the scene and performer pages from loading, mb